### DISCUSSION TITLE
Welcome to MIMWAL Discussions!

### DISCUSSION THREAD

**Discussion by NileshGhodekar** [microsoft/MIMWAL#95] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by JayVDZ** [comment]
This is a great addition @NileshGhodekar. 
Along with the usual discussions, I think it'll be a good place for folks to share how they're using MIMWAL.
